### PR TITLE
Hide backup code enable option in login flow in sub organizations

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [X] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/public/deployment.config.json [hunk 1]
```diff
@@ -1180,6 +1180,7 @@
             "applicationRequestPathAuthentication": false,
             "applicationSystemAppsSettings": false,
             "certificates": false,
+            "enablingBackupCodesForB2BUsers": true,
```
Comment (brionmario) [inline apps/console/src/public/deployment.config.json:1183]:
```suggestion
            "backupCodesForSubOrganizations": true,
```

Comment (brionmario) [reply apps/console/src/public/deployment.config.json:1183]: Validate if any `legacyMode` configs are there in `default.json` as well.

Comment (pavinduLakshan) [reply apps/console/src/public/deployment.config.json]: > Validate if any `legacyMode` configs are there in `default.json` as well.

legacyMode configs are not there in default.json.[1]

[1] https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

Comment (brionmario) [reply apps/console/src/public/deployment.config.json]: My bad, i meant `infer.json`

https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.infer.json#L247

Comment (pavinduLakshan) [reply apps/console/src/public/deployment.config.json]: Sent a PR

https://github.com/wso2/carbon-identity-framework/pull/5622

**Review by brionmario — APPROVED** [review]

File: features/admin.authentication-flow-builder.v1/components/nodes/sign-in-box-node/sign-in-box-node.tsx [hunk 2]
```diff
@@ -51,14 +51,20 @@ import EmailOTPFragment from "./fragments/email-otp-fragment";
...
+import useUIConfig from "../../../../admin.core.v1/hooks/use-ui-configs";
 import {
     IdentityProviderManagementConstants
 } from "../../../../admin.identity-providers.v1/constants/identity-provider-management-constants";
 import { GenericAuthenticatorInterface } from "../../../../admin.identity-providers.v1/models/identity-provider";
+import { useGetCurrentOrganizationType } from "../../../../admin.organizations.v1/hooks/use-get-organization-type";
 import useAuthenticationFlow from "../../../hooks/use-authentication-flow";
 import "./sign-in-box-node.scss";
 
+
```
Comment (pavinduLakshan) [inline features/admin.authentication-flow-builder.v1/components/nodes/sign-in-box-node/sign-in-box-node.tsx:67]:
```suggestion
```

**Review by brionmario — APPROVED** [review]

**Pull request merged**